demo/src/layout/IconDoc.js

```diff
@@ -70,28 +71,40 @@ class IconDoc extends Component<Props, State> {
         <div className={classes.container}>
           <Typography type="title">{`<Icon />`}</Typography>
           <Typography type="body">
-            In addition to all the Material Design Icons– we also ship a handful
+            In addition to all the Material Design Icons, we also ship a handful
             of custom icons.
           </Typography>
+          <Alert style={{ marginTop: "16px" }} color="info">
+            <Typography type="body" style={{ fontSize: "16px" }}>
+              Icons live in a separate Github repo {""}
+              <a
+                className={classes.link}
+                href="https://github.com/elm-street-technology/elevate-ui-icons"
+              >
+                here
+              </a>
+              .
+            </Typography>
+          </Alert>
           <LiveExample
             element={() => <IconExample classes={classes} />}
             code={`
-const IconExample = (props) => {
-  const { classes } = props;
-  return (
-    <div>
-      <div className={classes.iconContainer}>
-        <Icon name="AlarmOn" color="#55c3ba" size={34} />
-      </div>
-      <div className={classes.iconContainer}>
-        <Icon name="AddCircleOutline" color="rgb(77, 77, 77)" size={24} />
-      </div>
-      <div className={classes.iconContainer}>
-        <Icon name="Pool" color="#f15953" size={48} />
-      </div>
-    </div>
-  );
-};
+    const IconExample = (props) => {
+      const { classes } = props;
+      return (
+        <div>
+          <div className={classes.iconContainer}>
+            <Icon name="AlarmOn" color="#55c3ba" size={34} />
+          </div>
+          <div className={classes.iconContainer}>
+            <Icon name="AddCircleOutline" color="rgb(77, 77, 77)" size={24} />
+          </div>
+          <div className={classes.iconContainer}>
+            <Icon name="Pool" color="#f15953" size={48} />
+          </div>
+        </div>
+      );
+    };
           `}
           />
```

Comment:
Not in this diff, but lets add a `placeholder` to the search box input that has an example of an icon names. Something like `Edit, Add, etc...`